src/controllers/resource-store/index.tsx

```diff
@@ -100,9 +126,14 @@ export const actions: Actions = {
         return withExpiresAt;
       }
 
+      cached.accessedAt = getAccessedAt();
+      dispatch(setResourceState(type, key, cached));
+
       return cached;
     }
 
+    dispatch(validateLRUCache(resource, key));
```

Comment:
Can we move this inside `getResourceFromRemote`? I think it fits better there as cleanup is needed only right before adding the new `pending` resource (line 162).

Reply:
Done.

Reply:
✅